client/src/utils/blockstackHelpers.js

```diff
@@ -8,6 +8,17 @@ export const postSingleObservation = async (userSession, observation, observatio
   userSession.putFile(`observation/${fileNumber}.json`, JSON.stringify(observation));
 };
 
+export const deleteAllObservations = async userSession => {
+  await userSession.deleteFile(`observations.json`);
+};
+
+export const deleteSingleObservations = async (userSession, numOfObservations) => {
```

Comment:
I would suggest renaming deleteSingleObservations to something like deleteObservation. Only to clearly demonstrate that the method only deletes one observation. When I first looked at the method I was not sure if it deleted only one observation.

Reply:
The method can delete more than one observation.

Reply:
Ok I understand now that there is an observations file and there is a file for every single observation.

Reply:
Yeah so one thought about naming, for the clarity of future contributors around what these functions actually do... delete all observations definitely doesn't delete all observations... delete all and delete single sound like sort of do different operations on the same list... like delete a whole list vs delete one item out of a list.  So I'd suggest something like deleteObservationsList and deleteObservationFile.  But I also think that naming is going to be inherently a little confusing here for anyone who doesn't realize in advance that we have this little bit of data redundancy going on.

Reply:
Hm... the method deletes many observations in a loop, so I worry deleteObservation also wouldn't be an accurate name?